src/Command/DispatchCommand.php

```diff
@@ -169,7 +183,13 @@ private function updateLabels(Package $package): void
         $configuredLabels = $this->configs['labels'];
         $missingLabels = $configuredLabels;
 
-        $headers = ['Name', 'Actual color', 'Needed Color', 'State'];
+        $headers = [
+            'Name',
+            'Actual color',
+            'Needed Color',
+            'State'
```

Comment:
```suggestion
            'State',
```

Reply:
Yes, I am talking to @soullivaneuh why the GithubAction fails, but FlintCi is happy, so lets keep it as an example until we found out why! 👍

Reply:
IMO with all the checks we have now, we could consider removing flintCI.

Reply:
FlintCI creates PullRequests which are merged automatically, so we need to keep it until we fix, and commit the fixed files using a GithubAction. Lets postpone this a bit

Reply:
> FlintCI creates PullRequests which are merged automatically, so we need to keep it until we fix, and commit the fixed files using a GithubAction. Lets postpone this a bit

I may be wrong but the 3.x or master branch are not supposed to have lint-failures since we only merge PR with a green build. And if someone open a PR with a lint-failure, the build will be red and we'll ask for fixing it before merging it.

Or maybe it's only useful only when we're adding new rules to linters ?